.travis.yml

```diff
@@ -14,6 +14,9 @@ after_success: |
 env:
   global:
     secure: OiQj6NSyu634pS+qWCAeaibuBdiRsDMz5k0bi8hhdom3W24L8I9KkdWlBm2K8Ef97Uzgg6Xsr7LSWX0CONxX2eU3QaB1v3mXNxModa7ftW/w9y7B8MThxS5nRTUmHZC17OvoQ3EoNyrktnTAkyIr0E0YxBruJYAi4EH+54guRN8=
+  matrix:
+    - FEATURE=""
+    - FEATURE="v5"
```

Comment:
Unfortunately I think that this will cause the doc uploads to race with each other to see who wins. Perhaps just a new `cargo test --features v5` could be added?